src/styles/blocks/catalog.scss

```
@@ -0,0 +1,30 @@
+.catalog {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(1, $cardWidth);
```

no need to repeat 1 column
```suggestion
  grid-template-columns: $cardWidth;
```